src/compiler/diagnosticMessages.json

```diff
@@ -5311,6 +5283,58 @@
         "category": "Error",
         "code": 9006
     },
+    "Modules": {

```

Comment:
These should be in or near the original mesasges, instead of errors. So somewhere in 6000